lib/api/objectPutPart.js

```diff
@@ -179,12 +177,11 @@ function objectPutPart(authInfo, request, streamingV4Params, log,
                 });
         },
         (destinationBucket, objectLocationConstraint, cipherBundle,
-        splitter, next) => {
-            if (config.backends.data === 'multiple') {
-                return multipleBackendGateway.uploadPart(request,
-                streamingV4Params, null, size, objectLocationConstraint,
-                objectKey, uploadId, partNumber, bucketName, log,
-                (err, partInfo) => {
+         splitter, next) => {
+            const res = data.uploadPart(
```

Comment:
style: I don't think `res` needs to be returned since the processing is asynchronous, just calling `data.uploadPart(...);` should be good.